projects/angular/src/data/datagrid/all.spec.ts

```diff
@@ -72,6 +73,7 @@ describe('Datagrid', function () {
     DisplayModeServiceSpecs();
     TableSizeServiceSpec();
     ColumnResizerServiceSpecs();
+    ColumnsServiceSpec();
```

Comment:
Please pull this out to its own `chore` PR so we can get it merged now while we work on fixing the underlying scroll issue.